### PR TITLE
Updated docs to use ContentComponent instead of FormComponent.

### PR DESCRIPTION
**Pull request by jsiemensLatitudegeo**
The two pages modified can be viewed here:
https://deploy-preview-13--vertigis-studio-dev-center.netlify.app/docs/workflow/sdk-mobile-form-reference
https://deploy-preview-13--vertigis-studio-dev-center.netlify.app/docs/workflow/tutorial-mobile-activity-indicator-form-element/

### CODE REVIEW

File: docs/workflow/tutorial-mobile-activity-indicator-form-element.mdx [hunk 1]
```diff
@@ -32,31 +32,36 @@ Implementing a custom form element in VertiGIS Studio Mobile consists of three s
...
 ## Set up the Custom Form Element Skeleton
 
-1. Create a new file `LoadingIndicator.cs` in the platform agnostic project of the VertiGIS Studio Mobile [Quickstart](../mobile/sdk-overview.mdx).
-2. Add a new skeleton form element that implements `FormComponent`.
+1. Using Visual Studio, create a new file, using the Xamarin Forms: Content View template. Name the file `LoadingIndicator.xaml`, and place it in the platform agnostic project of the VertiGIS Studio Mobile [Quickstart](../mobile/sdk-overview.mdx).
+2. Change the base class to `ContentComponent` both in the xaml and in the code-behind.
+
+```xml title="App1/App1/workflow/LoadingIndicator.xaml"
+<?xml version="1.0" encoding="UTF-8"?>
+// highlight-next-line
```
Comment (brayden-marshall) [inline docs/workflow/tutorial-mobile-activity-indicator-form-element.mdx:40]: I don't think this is doing what you intended it to. For me, I just see the comment text appear as-is, it doesn't actually highlight the line in the rendered Markdown.

Same thing goes for the other usages later in the file.

**Review by brayden-marshall — CHANGES_REQUESTED** [review]

File: docs/workflow/tutorial-mobile-activity-indicator-form-element.mdx (hunk 1 above)
Comment (jsiemensLatitudegeo) [reply docs/workflow/tutorial-mobile-activity-indicator-form-element.mdx:40]: Looks correct to me. How are you viewing the file? Are you viewing the 'built' app, or just the raw mdx file?

image

File: docs/workflow/sdk-mobile-form-reference.mdx [hunk 2]
```diff
@@ -47,67 +55,64 @@ namespace App1.Workflow
 
 ## Using Form Elements
 
-Custom form elements can be used in a form through the special ["custom" form element](https://docs.vertigisstudio.com/workflow/latest/help/Default.htm#wf5/help/form-elements/custom.htm#Custom_Form_Elements%3FTocPath%3DForms%7CForm%2520Element%2520Reference%7CCustom%2520Form%2520Elements%7C_____0). This form element displays custom form elements by referencing them by the `id` they were registered with.
+Custom form elements can be used in a form through the special ["custom" form element](https://docs.geocortex.com/workflow/latest/help/Default.htm#wf5/help/form-elements/custom.htm#Custom_Form_Elements%3FTocPath%3DForms%7CForm%2520Element%2520Reference%7CCustom%2520Form%2520Elements%7C_____0). This form element displays custom form elements by referencing them by the `id` they were registered with.
```
[the remark this answers is not in the document]
Comment (jsiemensLatitudegeo) [reply docs/workflow/sdk-mobile-form-reference.mdx:58]: Oh yes, that's a mistake. I originally cloned the repo from the geocortex org, then was told by Ryan that it had moved to vertigis. I copied the file that I had changed into the new repo, without updating the links. Will fix.

File: docs/workflow/tutorial-mobile-activity-indicator-form-element.mdx (hunk 1 above)
Comment (brayden-marshall) [reply docs/workflow/tutorial-mobile-activity-indicator-form-element.mdx:40]: Oh, my bad. I was viewing the Markdown as rendered by Github.

File: docs/workflow/sdk-mobile-form-reference.mdx [hunk 3]
```diff
@@ -121,27 +126,41 @@ namespace App1.Workflow
 
 A custom form element may produce a value that a workflow needs to access at runtime. You can set a property on the `this.element` object to expose that value to the form.
```
Comment (Felicity-R) [inline docs/workflow/sdk-mobile-form-reference.mdx:127]:
```suggestion
A custom form element may produce a value that a workflow needs to access at runtime. You can set a property on the `this.Element` object to expose that value to the form.
```

File: docs/workflow/sdk-mobile-form-reference.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jsiemensLatitudegeo) [reply docs/workflow/sdk-mobile-form-reference.mdx]: Fixed.

File: docs/workflow/sdk-mobile-form-reference.mdx (hunk 3 above)
Comment (jsiemensLatitudegeo) [reply docs/workflow/sdk-mobile-form-reference.mdx]: Fixed.

File: docs/workflow/tutorial-mobile-activity-indicator-form-element.mdx [hunk 4]
```diff
@@ -164,35 +169,21 @@ namespace App1.Workflow
...
+<?xml version="1.0" encoding="UTF-8"?>
+<core:ContentComponent xmlns="http://xamarin.com/schemas/2014/forms"
+                       xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                       xmlns:core="clr-namespace:VertiGIS.Mobile.Workflow.Core;assembly=VertiGIS.Mobile.Workflow"
+                       // highlight-next-line
+                       xmlns:views="clr-namespace:VertiGIS.Mobile.Toolkit.Views;assembly=VertiGIS.Mobile.Toolkit"
+                       x:Class="App1.Workflow.LoadingIndicator">
+    <ContentView.Content>
+        // highlight-next-line
+        <views:EnhancedActivityIndicator IsEnabled="{Binding IsEnabled}" IsRunning="True" HeightRequest="75" WidthRequest="75" Margin="10"/>
```
[the remark this answers is not in the document]
Comment (jsiemensLatitudegeo) [reply docs/workflow/tutorial-mobile-activity-indicator-form-element.mdx:184]: Why?

[the remark this answers is not in the document]
Comment (Felicity-R) [reply docs/workflow/tutorial-mobile-activity-indicator-form-element.mdx:184]: Isn't the name of the custom element that we created just "LoadingIndicator"? I don't see the name "EnhancedActivityIndicator" anywhere else in the file so it seems wrong, unless I'm missing something...

[the remark this answers is not in the document]
Comment (jsiemensLatitudegeo) [reply docs/workflow/tutorial-mobile-activity-indicator-form-element.mdx:184]: It's referencing a the toolkit view: EnhancedActivityIndicator, not the form component.

**Review by Felicity-R — APPROVED** [review]

**Review by rcooney — APPROVED** [review]

**Pull request merged**